addons/mail/static/src/widgets/form_renderer/form_renderer.js

```diff
@@ -15,6 +15,9 @@ class ChatterContainerWrapperComponent extends ComponentWrapper {}
  * subset of) the mail widgets (mail_thread, mail_followers and mail_activity).
  */
 FormRenderer.include({
+    events: Object.assign({}, FormRenderer.prototype.events, {
+        'touchmove': '_onScrollForm', // Mobile
+    }),
```

Comment:
Why? Doesn't a touch event that lead to a scroll change also generate a scroll event?